Remotion/Data/DomainObjects/Queries/Configuration/QueryDefinition.cs

```diff
@@ -76,6 +78,9 @@ public class QueryDefinition
   /// <param name="queryType">
   /// One of the <see cref="QueryType"/> enumeration constants.
   /// </param>
+  /// <param name="metaData">
+  /// Adds metadata to the query.
```

Comment:
What is the intended use for the metadata? This comment should explain to the reader why he might want to use the parameter.